src/Javis.jl

```diff
@@ -550,6 +551,9 @@ function get_javis_frame(video, objects, frame; layers = Layer[])
         end
 
         img_layers = place_layers(video, layers, frame)
+        for (position, layer) in zip(starting_positions, layers)
```

Comment:
Maybe just a comment 
`# resetting layer positions to originals` or something like that